dashboard/job/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "project_id" { type = string }
+variable "job_name" { type = string }
+
+variable "alert_policies" {
+  type = map(object({
+    id = string
+  }))
+  default = {}
```

Comment:
I've been trying to add detailed comments to the `variables` and `outputs` so our auto-docs are 💯

Reply:
I've also been trying to add fairly decent usage comments to the `README.md` files.

Reply:
Can I give you an IOU for the alert policy stuff for now? That's my next chunk of work and I might completely redo how it's structured.